vllm/model_executor/models/aria.py

```diff
@@ -508,12 +575,14 @@ def input_processor(ctx, llm_inputs):
         prompt_token_ids=new_token_ids,
         prompt=new_prompt,
         multi_modal_data={"image": image_inputs},
+        multi_modal_placeholders={"image": ranges},
     )
 
 
-@MULTIMODAL_REGISTRY.register_max_image_tokens(get_max_multimodal_tokens)
+@MULTIMODAL_REGISTRY.register_max_image_tokens(get_aria_max_multimodal_tokens)
 @MULTIMODAL_REGISTRY.register_image_input_mapper(input_mapper_for_aria)
-@INPUT_REGISTRY.register_input_processor(input_processor)
+@INPUT_REGISTRY.register_dummy_data(dummy_data_for_aria)
```

Comment:
The code for dummy data generation was entirely missing and I'm not sure why, so I added for this model since it's required for V1. cc @xffxff who originally added this model